src/components/verification/VerificationCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { VerificationStatus } from "@/types/types";
+import { Box, Text } from "@ledgerhq/react-ui";
+import {
+  CircledCrossSolidMedium,
+  InfoAltFillMedium,
+  WarningSolidMedium,
+} from "@ledgerhq/react-ui/assets/icons";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "styled-components";
+
+type Props = {
+  verification: VerificationStatus;
+  [key: string]: unknown;
+};
```

Comment:
We can improve the typings here using `React.CSSProperties`
```suggestion
type Props = {
  verification: VerificationStatus;
} & React.CSSProperties;
```